tests/foreman/cli/test_import.py

```diff
@@ -66,7 +67,9 @@ def build_csv_file(rows=None, dirname=None):
         )
         csv_writer.writeheader()
         for row in rows:
-            csv_writer.writerow(row)
+            csv_writer.writerow({
+                key: val.encode('utf8') for key, val in row.items()
```

Comment:
Using dict comprehension we are explicitly saying that we does not support python 2.6. I completely agree on this :smile: